gm4_liquid_minecarts/data/liquid_minecarts/functions/compare_load_liquid.mcfunction

```diff
@@ -0,0 +1,8 @@
+#@s = liquid minecart containing a liquid with a liquid tank containing liquid pointed at it
+#run from try_to_load
+
+summon area_effect_cloud ~ ~ ~ {Tags:["gm4_liquid_minecart_compare"],Duration:1}
+data modify entity @e[type=area_effect_cloud,limit=1,sort=nearest,tag=gm4_liquid_minecart_compare] CustomName set from block ~ ~ ~ CustomName

```

Comment:
I worry that this won't be a safe way to compare what type of liquid the tank when things are moved to translation keys, though I'm not certain. Thoughts?

Reply:
I don't know, we'd have to test I guess. Tehnically the tank should copy the translation string and compare that, right?

There isn't really any other way to make this module work with ANY liquid without modifying liquid tanks to support this one.

Reply:
This brings up a more general question:
Should we scrap concepts / introduce extra checks just so an optional language pack works?

Reply:
If GM4 was chinese and I wanted to use it and some of the modules had english translations... I'd rather have chinese liquid Minecarts I can work out how to use than no liquid minecarts at all.

Reply:
Oh, wait. This os a glaring bug. In 1.14 we were planning to remove any tank name checks from Liquid Tanks as a user can quickly swap them with a different one, which now just locks some of the functionality. 

The check has to be done on the tags so maybe when you copy the data, you should store the tags list into an nbt tag.

Reply:
This seems to be an issue according to @Kroppeb. I don't understand how liquid tanks works but is it fixable? Should we try to fix it?

Reply:
I was able to confirm this bug actually works.
Here are the steps:
## How to convert liquid types:
* Have a liquid tank of the desired type `desired` and one with the liquid you want to use `current`
* Move liquid from `desired` to a liquid minecart `LM` but make sure there is still some liquid left in `desired`
* Break `desired` (and replace with a different hopper if you want to keep the liquid)
* Break `current` and quickly replace with the hopper you got from breaking `desired`
* Move `LM` below `current`

Reply:
Does breaking a hopper with json- CustomName really transfer its name to the item properly? Or is this only applicable to tanks without translation keys?

Reply:
Before executing the bug, I fetched this branch and your liquid tanks branch and merged them and tested in that state. I did forcefully make the ml take the liquid out due to a bug that now *should* be fixed. Even if that caused it and breaking json tanks doesn’t work, one can still reset the name before transferring the liquid to and from tanks

Reply:
So.... this is fixed because the bug you say *should* be fixed?

Reply:
> So.... this is fixed because the bug you say _should_ be fixed?

No, bug I'm referencing was the issue that minecarts weren't filled from south and north IIRC

Reply:
I do have a solution in mind, will make a pull for that later maybe.